WordPress-iOS-Shared.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "WordPress-iOS-Shared"
-  s.version      = "0.6.3"
+  s.version      = "0.6.4"
```

Comment:
Actually, thinking about this now – should this be a larger version update (i.e. not just incrementing the final digit)? It's a backwards-incompatible update, as we're removing functionality?